release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -3383,3 +3383,27 @@ $ oc adm release info 4.9.55 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-56"]
+=== RHSA-2023:0778 - {product-title} 4.9.56 security update
+
+Issued: 2023-02-22
+
+{product-title} release 4.9.56, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2023:0778[RHSA-2023:0778] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:0777[RHSA-2023:0777] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.9.56 --pullspecs
+----
+
+[id="ocp-4-9-56-bug-fixes"]
+==== Bug fixes
+
+* Previously, pod failures were artificially extending the validity period of certificates causing them to incorrectly rotate. With this update, the certificate validity period is correctly determined and the certificates are correctly rotated. (link:https://issues.redhat.com/browse/OCPBUGS-5938[*OCPBUGS-5938*])
```

Comment:
just a suggestion, up to you if you want any of it. I was thinking more along the lines of past/present tense here:
```suggestion
* Previously, pod failures artificially extended the validity period of certificates causing them to incorrectly rotate. With this update, the certificate validity period is correctly determined and certificates rotate correctly. (link:https://issues.redhat.com/browse/OCPBUGS-5938[*OCPBUGS-5938*])
```
The only thing I am wondering is if we can use another word for correctly for one of the instances in the last sentences. There are 2 and it trips me up a bit.

Reply:
Yeah I think that would work. Thanks!

Reply:
@opayne1 Thank you!